src/ui/src/builder/BuilderModelSelect.vue

```diff
@@ -0,0 +1,95 @@
+<script setup lang="ts">
+import injectionKeys from "@/injectionKeys";
+import {
+	computed,
+	defineAsyncComponent,
+	inject,
+	onMounted,
+	PropType,
+} from "vue";
+import { useListResources } from "@/composables/useListResources";
+import type { Option } from "@/wds/WdsSelect.vue";
+import BuilderAsyncLoader from "./BuilderAsyncLoader.vue";
+import type { WriterGraph } from "@/writerTypes";
+import WdsTextInput from "@/wds/WdsTextInput.vue";
+
+const WdsSelect = defineAsyncComponent({
+	loader: () => import("@/wds/WdsSelect.vue"),
+	loadingComponent: BuilderAsyncLoader,
+});
+
+const wf = inject(injectionKeys.core);
+
+defineProps({
+	enableMultiSelection: { type: Boolean, required: false },
+});
+
+const currentValue = defineModel({
+	type: [String, Array] as PropType<string | string[]>,
+	required: true,
+});
+
+const {
+	load: loadGraphs,
+	data: graphs,
+	isLoading,
+} = useListResources<WriterGraph>(wf, "models");
```

Comment:
You use the wrong type here (it works though because `WriterGraph` has `name` & `id`).

You should use introduce a new type `WriterModel` like this one

https://github.com/writer/writer-framework/blob/117920a0b05b13a17ad2a89138238f0d7c15a8e7/src/ui/src/writerTypes.ts#L212-L224

and use it here (with taking care of renaming vars names)
```suggestion
const {
	load: loadModel,
	data: models,
	isLoading,
} = useListResources<WriterModel>(wf, "models");
```